src/mon/Monitor.cc

```diff
@@ -3762,14 +3762,14 @@ void Monitor::handle_ping(MonOpRequestRef op)
   MPing *reply = new MPing;
   entity_inst_t inst = m->get_source_inst();
   bufferlist payload;
-  Formatter *f = new JSONFormatter(true);
```

Comment:
@xiexingguo could you file a ticket for this change? so we can backport the fix to jewel?